README.md

```diff
@@ -1,4 +1,20 @@
-Travis-CI: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
-
 cob_android
 ===========
+
+## ROS Distro Support
+
+         | Indigo | Jade | Kinetic
+-------- | ------ | ---- | -------
+ Branch  | [`indigo_dev`](https://github.com/ipa320/cob_android/tree/indigo_dev) | [`indigo_dev`](https://github.com/ipa320/cob_android/tree/indigo_dev) | [`indigo_dev`](https://github.com/ipa320/cob_android/tree/indigo_dev) |
+ Status  |  supported | not supported |  supported |
+ Version | [version](http://repositories.ros.org/status_page/ros_indigo_default.html?q=cob_android) | [version](http://repositories.ros.org/status_page/ros_jade_default.html?q=cob_android) | [version](http://repositories.ros.org/status_page/ros_kinetic_default.html?q=cob_android) |
+
+## Travis - Continuous Integration
+
+Status: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
+
+## ROS Buildfarm
+
+        | Indigo Source | Indigo Debian | Jade Source | Jade Debian |  Kinetic Source  |  Kinetic Debian |
+------- | ------------------- | ------------------- | ------------------- | ------------------- | ------------------- | -------------------
+cob_android | [![not released](http://build.ros.org/buildStatus/icon?job=Isrc_uT__cob_android__ubuntu_trusty__source)](http://build.ros.org/view/Isrc_uT/job/Isrc_uT__cob_android__ubuntu_trusty__source/) | [![not released](http://build.ros.org/buildStatus/icon?job=Ibin_uT64__cob_android__ubuntu_trusty_amd64__binary)](http://build.ros.org/view/Ibin_uT64/job/Ibin_uT64__cob_android__ubuntu_trusty_amd64__binary/) | [![not release](http://build.ros.org/buildStatus/icon?job=Jsrc_uT__cob_android__ubuntu_trusty__source)](http://build.ros.org/view/Jsrc_uT/job/Jsrc_uT__cob_android__ubuntu_trusty__source/) | [![not release](http://build.ros.org/buildStatus/icon?job=Jbin_uT64__cob_android__ubuntu_trusty_amd64__binary)](http://build.ros.org/view/Jbin_uT64/job/Jbin_uT64__cob_android__ubuntu_trusty_amd64__binary/) | [![not release](http://build.ros.org/buildStatus/icon?job=Ksrc_uX__cob_android__ubuntu_xenial__source)](http://build.ros.org/view/Ksrc_uX/job/Ksrc_uX__cob_android__ubuntu_xenial__source/) | [![not release](http://build.ros.org/buildStatus/icon?job=Kbin_uX64__cob_android__ubuntu_xenial_amd64__binary)](http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__cob_android__ubuntu_xenial_amd64__binary/) |
```

Comment:
minor issue, but you have written `not release` instead of `not released` for all but the first two :-)